examples/capture/main.rs

```diff
@@ -108,14 +127,35 @@ async fn run() {
     }
 
     if let Ok(()) = buffer_future.await {
-        let data = output_buffer.get_mapped_range(0, wgt::BufferSize::WHOLE);
-        let mut png_encoder = png::Encoder::new(File::create("red.png").unwrap(), size, size);
+        let padded_buffer = output_buffer.get_mapped_range(0, wgt::BufferSize::WHOLE);
+        // There are two transforms occuring here:
+        // 1.   map bgra -> rgba
+        // 2.   remove padding from buffer
```

Comment:
When generating a 1000x1000 image, using Rgba + stream_writer was ~5ms faster then my old implementation.
Assuming the user is rendering in a headless mode i.e. not copying from the swapchain.
Then theres nothing stopping them from configuring their pipelines etc. to use rgba instead of bgra when rendering to an image.
So using rgba + stream_writer seems like a good idea!